### DISCUSSION TITLE
RFC: PubPub's Updated Terms, Privacy Policy, and Acceptable Use Policy

### DISCUSSION THREAD

**Discussion by gabestein** [knowledgefutures/pubpub#1248] [General]
We're building PubPub for readers, publishers, and knowledge creators. Not for analytics companies. We've been working hard with a legal team that specializes in open-source software on updating PubPub's Privacy Policy, Terms of Service, and Acceptable Use Policy to be easier to understand, more comprehensive, and more protective of user and community rights on PubPub, while also trying to minimize PubPub's rights as much as possible.

You can read the proposed new terms at the following URLs:

https://www.pubpub.org/legal/proposed-terms
https://www.pubpub.org/legal/proposed-privacy
https://www.pubpub.org/legal/proposed-aup

These terms will replace our existing terms and go into effect two weeks from now, on Monday, February 8, 2021. In the meantime, we sincerely welcome your feedback and comments on our proposed terms.

**Comment by mapc088** [comment]
The three documents are really clear and easy to understand. I really like the acceptable use policy, which is very specific. I just want to ask why you decided to include under the US restrictions the non-US user's agreement? I think the consequence is clear, in particular to European users, which at the moment, have a more strict privacy policy than the US. 

Thanks for the transparency and quality, as always :-)

**Reply by gabestein** [reply]
Thanks for the feedback, and the question! To be clear, are you referring to 7.2 of the Privacy Policy?

**Reply by mapc088** [reply]
My apologies for the late reply.
Yes, it is 7.2

**Reply by gabestein** [reply]
No worries! I think that part is one of the most confusing parts of the Privacy Policy. To be clear: our intent is to provide the strictest privacy protection possible. I believe we are both GDPR and CCPA compliant.

As I understand it, that section relates to the jurisdictional authority of the United States. We operate our servers in US regions (for now), which has different laws than the EU when it comes to the relationship between service providers and government.

**Comment by gabestein** [comment]
Just a note that these terms have been formally adopted and are visible at https://pubpub.org/legal
